app/src/main/java/com/example/android/codelabs/navigation/fragments/DeepLinkFragment.kt

```diff
@@ -45,15 +45,13 @@ class DeepLinkFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
-        val textView = view.findViewById<TextView>(R.id.text)
-        textView.text = arguments?.getString("myarg")
+        text_deeplink.text = arguments?.getString("myarg")
```

Comment:
Pls, use a const or a TAG for this hardcode

Reply:
done